dds_web/api/superadmin_only.py

```diff
@@ -163,6 +163,20 @@ def post(self):
         if not motd_obj or not motd_obj.active:
             raise ddserr.DDSArgumentError(message=f"There is no active MOTD with ID '{motd_id}'.")
 
+        # check if sent to unit personnel only or all users
+        unit_personnel_only: bool = request_json.get("unit_personnel_only", False)
+        if not isinstance(unit_personnel_only, bool):
+            raise ddserr.DDSArgumentError(
+                message="The 'unit_personnel_only' argument must be a boolean."
+            )
```

Comment:
As mentioned in the CLI PR, `unit_personnel_only` sounds like we're only sending to `Unit Personnel`, and not also `Unit Admin`s. Should be changed to `unit users` in this case.